python/vendor/hive_metastore/ThriftHiveMetastore-remote

```diff
@@ -0,0 +1,3364 @@
+#!/usr/bin/env python
+#
+# Autogenerated by Thrift Compiler (0.16.0)
+#
+# DO NOT EDIT UNLESS YOU ARE SURE THAT YOU KNOW WHAT YOU ARE DOING
+#
+#  options string: py
+#
+
+import pprint
+import sys
+
+if sys.version_info[0] > 2:
+    from urllib.parse import urlparse
+else:
+    from urlparse import urlparse
+
+from hive_metastore import ThriftHiveMetastore
+from hive_metastore.ttypes import *
+from thrift.protocol.TBinaryProtocol import TBinaryProtocol
+from thrift.transport import (
+    THttpClient,
+    TSocket,
+    TSSLSocket,
+    TTransport,
+)
+
+if len(sys.argv) <= 1 or sys.argv[1] == "--help":
```

Comment:
Is this going to run when `hive_metastore` is imported? Since it assumes that this is the main script and calls `sys.exit(0)`, we may want to edit it.

Reply:
No, we're not using this at all. It is autogenerated and we get this for free, but I don't think anyone would use this. I would just leave it in, since it isn't being imported anywhere.

Reply:
I think we should remove it if it isn't needed. It's just unnecessary code to deal with. We only need to exclude it the first time. The next time we generate this only the files that were already added will be changed on commit.